Source/Foundation/bsfUtility/Utility/BsOctree.h

```diff
@@ -331,7 +325,7 @@ namespace bs
 			/** Calculates bounds for the provided child node. */
 			NodeBounds getChild(HChildNode child) const
 			{
-				static constexpr float map[] = { -1.0f, 1.0f };
+				static constexpr const float map[2] = { -1.0f, 1.0f };
```

Comment:
Is there an advantage of using const here? Also the specific statement of array size seems redundant perhaps.

Reply:
We can drop this changed line, that's no problem.

I'm a fan of specificity when it comes to array sizes, but given it's only 2 items I understand why most people wouldn't care to bother.

As for the constexpr vs const, I'm not 100% certain that there is or isn't a difference. My understanding of the standard is that constexpr variable are still modifiable in some situations in compile-time contexts, but to be honest I haven't put in the due-diligence to actually verify that one way or another, and I don't know of any examples where that could happen.

Feel free to drop this change.